app/scripts/controllers/newOverview.js

```diff
@@ -1195,13 +1197,13 @@ function OverviewController($scope,
       group: "extensions",
       resource: "deployments"
     }, context, function(deploymentData) {
-      overview.deployments = deploymentData.by('metadata.name');
+      overview.deploymentsByUID = deploymentData.by('metadata.uid');
```

Comment:
this isnt new, but were we not sorting deployments before?  how is it being ordered when you are looking at the by Resource Type view?